src/internal/connector/exchange/api/contacts.go

```diff
@@ -55,6 +55,41 @@ func (c Contacts) CreateContactFolder(
 	return mdl, nil
 }
 
+// GetContactFolderByDisplayName fetches contact folder which has the displayName of
```

Comment:
I'd love to see a gock-mocked integration test for this.  The mocks should help to avoid external data dependencies, making it fairly easy to inject.

Reply:
sgtm